src/client/components/DeliveryInfo/DeliveryInfo.component.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './DeliveryInfo.styles.css';
+
+export default function DeliveryInfo({ editMode, vertDisplay, user }) {
+  const [edit, setEdit] = React.useState(false);
+  const handleEdit = () => setEdit(!edit);
+  const handleSubmit = (event) => {
+    fetch('/api/users', {
+      method: 'post',
+      headers: {
+        Accept: 'application/json, text/plain, */*',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        address: event.target.address.value,
+        zipcode: event.target.zipcode.value,
+        city: event.target.city.value,
+        country: event.target.country.value,
+      }),
+    }).then((response) => {
+      if (response.ok) {
+        alert('User saved Successfully');
+      } else {
+        throw new Error(response.status);
+      }
+    });
+  };
+  return (
+    <div className="infoBox">
+      <div className="wrapper">
+        {edit ? (
+          <form className="form" onSubmit={handleSubmit}>
+            <label className="title">DELIVERY INFO</label>
+            <label className="addressInputLabel">Address:</label>
+            <input
+              defaultValue={user.address}
+              className="addressInput"
+              name="address"
+              type="text"
+            />
+
+            <label className="cityInputLabel">City:</label>
+            <input
+              defaultValue={user.city}
```

Comment:
Given that you destructured the user as suggested above you can access city directly:

```suggestion
              defaultValue={city}
```

Reply:
Changes made as per review comments